tests/api2/test_305_container.py

```diff
@@ -533,7 +531,23 @@ def test_40_verify_ix_chart_config_hostPathVolumes(request):
     assert payload_hostPathVolumes in config['hostPathVolumes'], results.text
 
 
-def test_41_delete_truecommand_chart_release(request):
+def test_41_verify_ix_chart_resources_host_path_volumes(request):
+    depends(request, ['tc_hostPathVolumes'])
+    payload = {
+        'query-options': {
+            'extra': {
+                'retrieve_resources': True
+            }
+        },
+        'query-filters': [['id', '=', tc_release_id]]
+    }
+    results = GET('/chart/release/', payload)
+    host_path_volumes = str(results.json()[0]['resources']['host_path_volumes'])
+    assert f'/mnt/{pool_name}/tc-hostpath' in host_path_volumes, host_path_volumes
+    assert f'/mnt/{pool_name}/ix-applications/releases/truecommand/volumes/tc-mountpath' in host_path_volumes, host_path_volumes
```

Comment:
`mountPath` is not available in `host_path_volumes`. It would be available in `pods`

Reply:
This only does not fail because this path is being consumed by an ix volume which has been specified in test 37.

Reply:
Yeah now it make sence